codec-http2/src/main/java/io/netty/handler/codec/http2/Http2GoAwayFrame.java

```diff
@@ -46,6 +49,11 @@
     Http2GoAwayFrame setExtraStreamIds(int extraStreamIds);
 
     /**
+     * Returns the last stream identifier if set, or {@code -1} else.
```

Comment:
I now see the comments at the class level (I just jumped to this method in the IDE when I made the comment) ... maybe this is good enough